src/data/customer/__test__/driver.test.js

```diff
@@ -10,6 +10,9 @@ import {
   deleteDriverScheduleAsync,
   createDriverScheduleAsync,
   updateDriverScheduleAsync,
+  getDriversWithFiltersAsync,
```

Comment:
@vdanylov please have a look at this PR from @vinay3206: https://github.com/CarPalFleet/JavaScript-SDK/pull/120
You might want to consider merging his code in your branch and re-run the test suites.

Reply:
@maartenhemmes yeap, we did it in that way, thanks)